addons/mail/models/mail_tracking_value.py

```diff
@@ -44,6 +44,26 @@ def _compute_field_groups(self):
             field = model._fields.get(tracking.field.name)
             tracking.field_groups = field.groups if field else 'base.group_system'
 
+    def _tracking_value_format(self):
+        tracking_values = []
+        for tracking in self:
+            groups = tracking.sudo().field_groups
```

Comment:
Yet another sudo. Seems like you are sudo-ing everything everytime because you have some hidden issues maybe ?

Reply:
`field_groups` used to be read form `message_sudo.tracking_value_ids` before the move, which was always sudo. So it makes sense to add it here in regards to the diff of this PR.

But I agree the need for this code is not very clean.